lldb/source/Target/Language.cpp

```diff
@@ -542,9 +542,22 @@ Language::Language() = default;
 // Destructor
 Language::~Language() = default;
 
+static llvm::dwarf::SourceLanguage
+ToDwarfSourceLanguage(lldb::LanguageType language_type) {
+  if (language_type < lldb::eLanguageTypeLastStandardLanguage)
+    return static_cast<llvm::dwarf::SourceLanguage>(language_type);
+
+  switch (language_type) {
+  case eLanguageTypeMipsAssembler:
+    return llvm::dwarf::DW_LANG_Mips_Assembler;
+  default: break;
+  }
+
+  llvm_unreachable("Unhandled language type");
```

Comment:
Are we sure we cannot get here even if. we parse DWARF with e.g., a DW_LANG_user_low+n language attribute?

Reply:
Otherwise a fail-safe that doesn't crash might be more appropriate.